src/analytics/Events.tsx

```diff
@@ -111,9 +111,9 @@ export enum KeylessBackupEvents {
   cab_issue_sms_code_error = 'cab_issue_sms_code_error',
   cab_enter_phone_code_back = 'cab_enter_phone_code_back',
   cab_enter_phone_code_cancel = 'cab_enter_phone_code_cancel',
-  cab_issue_valora_keyshare_start = 'cab_issue_valora_keyshare_start',
-  cab_issue_valora_keyshare_success = 'cab_issue_valora_keyshare_success',
-  cab_issue_valora_keyshare_error = 'cab_issue_valora_keyshare_error',
+  cab_issue_app_keyshare_start = 'cab_issue_app_keyshare_start',
+  cab_issue_app_keyshare_success = 'cab_issue_app_keyshare_success',
+  cab_issue_app_keyshare_error = 'cab_issue_app_keyshare_error',
```

Comment:
this is a non BC change, as it results in creating new analytics events and has analytics implications as we'll have to look at different events for different app versions. Also, depending on how we split out CAB, this maybe continued to be issued by Valora or by individual apps, so this event name maybe slightly misleading. Both are probably fine, but just wanted to call it out

Reply:
oh shoot, yeah i made a note of this as i made this change, but this was a big PR so i forgot about it by the time i wrote the description. i can update the description.

re: naming, yeah, we could probably keep the existing naming if we wanted, since our current plan is to share the CAB system between all MS apps (e.g., you can backup in one and restore in another), so calling it the e.g., "Valora CAB" system might be fine.

Reply:
i'm not sure if these events are being used for marketing segmentation in clevertap or on mixpanel dashboards, if we want to make this non BC change can you let the product / marketing folks know about the change @jophish ?

Reply:
https://valora-app.slack.com/archives/C025N1VU657/p1722970764905039